src/translations/en.json

```
@@ -987,6 +987,10 @@
           "settings": {
             "title": "Settings",
             "description": "Basic settings, server controls, logs and info"
+          },
+          "developer_tools": {
+            "title": "Developer Tools",
+            "description": "Try things out quickly"
```

People are often scared of the word "developer", so let's describe it better here
```suggestion
            "description": "Tools to help create automations and scripts"
```